client/my-sites/email/email-providers-comparison/index.jsx

```diff
@@ -523,7 +536,7 @@ class EmailProvidersComparison extends Component {
 			'email-providers-comparison__highlight-main-price': showBlackFridaySale,
 			'email-providers-comparison__keep-main-price': ! isEligibleForFreeTrial,
 		} );
-		const formattedPrice = translate( '{{price/}} /mailbox /month billed monthly', {
+		const formattedPrice = translate( '{{price/}} /mailbox /month (billed monthly)', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.